### DISCUSSION TITLE
2 x Starlink but the Speed is only 1 x Starlink in Bonding

### DISCUSSION THREAD

**Discussion by mastersteve** [Ysurac/openmptcprouter#3522] [Q&A]
I have 2 x Starlink connected with an N100 Mini PC. The VPS is AVORA. 
All works fine. But if i do an Speedtest it is like the Speed is from one Starlink connection and not from both.

If i unplug one connection from eth1 or eth2 it is the same speed.
Can somebody help me please.

Thank´s Steve

![Screenshot 2024-09-05 074259](https://github.com/user-attachments/assets/0aa51e94-c0a1-47e5-a023-9759fe715477)
![Screenshot 2024-09-05 074436](https://github.com/user-attachments/assets/85bf6329-fbb6-4755-a3a6-99032f8eb7a1)

**Comment by Ysurac** [comment]
Did you check in Network->MPTCP, "Bandwidth" tab if aggregation is working during a speed test ?
You can also use, via SSH on the router, ```omr-test-speed eth1``` and ```omr-test-speed eth2``` at the same test to check if both starlink give a good speed at same time and then ```omr-test-speed``` to check aggregation (Ctrl+C to stop a test after 2 minutes).

**Comment by mastersteve** [comment]
I have done a speedtest at eth1 and eth2. It is the half speed on eth1.
![Screenshot 2024-09-05 085848](https://github.com/user-attachments/assets/78cd31f2-7785-4e8e-b318-a4dadc7ec9c7)
![Screenshot 2024-09-05 090029](https://github.com/user-attachments/assets/7a25aa28-db96-40a5-b2ee-9e2fdc1daa2d)
![Screenshot 2024-09-05 090221](https://github.com/user-attachments/assets/bd433519-a0ae-4938-bac9-4198f32e7c89)
